builder-support/dockerfiles/Dockerfile.rpmbuild

```diff
@@ -1,8 +1,6 @@
 FROM dist-base as package-builder
 RUN touch /var/lib/rpm/* && \
-    yum install -y rpm-build rpmdevtools /usr/bin/python3 && \
-    yum groupinstall -y "Development Tools" && \
-    rpmdev-setuptree
+    yum install -y rpm-build rpmdevtools python3 "@Development Tools"
```

Comment:
@pieterlexis any opinion on this removal of `rpmdev-setuptree` ?

Reply:
It was called twice; once here, and once after setting WORKDIR.

Reply:
:+1: